bindings/python/python/bleuscore/__init__.py

```
@@ -1,6 +1,6 @@
-from .bleuscore import *
+from ._bleuscore import *
 
-__doc__ = bleuscore.__doc__
+__doc__ = _bleuscore.__doc__
```

Pylint (3.3.7)

[error] Undefined variable '_bleuscore'

(E0602)